tools/changelog-to-html.py

```diff
@@ -164,6 +159,17 @@ def makeCmdParser():
 def main(args):
 	args = makeCmdParser();
 
+	if len(missing_modules)>0: 
+		print_error("Warning: Could not generate HTML changelog. Missing python modules:")
+		for k,v in missing_modules.items():
+			print_error("    {:20s} {}".format(k,v))	
+		print_info("    To install run: 'pip install {}'".format(" ".join(missing_modules.keys())))
```

Comment:
I would still argue for
Python -m pip install 
since pip install requires the script path to be in the PATH on windows
https://stackoverflow.com/questions/34247930/pip-not-working-in-python-3-5-on-windows

Reply:
fixed